apl-core/src/main/java/com/apollocurrency/aplwallet/apl/exchange/model/DexTradeEntry.java

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright © 2018-2019 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.apl.exchange.model;
+
+import com.apollocurrency.aplwallet.api.dto.DexTradeInfoDto;
+import java.math.BigDecimal;
+
+/**
+ * Data class for interaction with trade table in the database
+ * @author Serhiy Lymar
+ */
+public class DexTradeEntry {
```

Comment:
Maybe better to use lombok?

Reply:
I can't agree more. It's already done, thnx